spring/boot1-autoconfigure/build.gradle

```diff
@@ -1,19 +1,24 @@
 final def SPRING_BOOT_VERSION = '1.5.22.RELEASE'
-final def MICROMETER_VERSION = '1.3.6'
+final def MICROMETER_VERSION = '1.5.0'
 
 dependencies {
     // To let a user choose between thrift and thrift0.9.
     compileOnly project(':thrift')
     implementation project(':logback')
 
-    [ 'micrometer-core', 'micrometer-spring-legacy' ].each {
-        implementation("io.micrometer:$it") {
-            version {
-                // Will fail the build if the override doesn't work
-                strictly MICROMETER_VERSION
-            }
+    implementation("io.micrometer:micrometer-spring-legacy") {
```

Comment:
@shakuzen Any thoughts on mixing LTS versions like this?

Reply:
It's probably better to keep the version consistent within this sub-project. I would expect a user still using `micrometer-spring-legacy` to stay on Micrometer 1.3.x. Of course the best suggestion is that they upgrade to Spring Boot 2 to be on the latest supported versions, though.

Reply:
Hmm - in that case can we backport the `ExecutorServiceMetrics` change to 1.3.x? Right now we have the conundrum of having it only in 1.5.x, but only a 1.3.x of spring-legacy.